scout/constants/clinvar.py

```diff
@@ -1,3 +1,4 @@
+CLINVAR_API_URL = "https://submit.ncbi.nlm.nih.gov/api/v1/submissions/?dry-run=true"
```

Comment:
This is not the actual clinvar submission ULR, whis is the same minus the `?dry-run=true`. I wrote it like this for testing purposes, but the URL should be fixed before merging